custom_components/velux/cover.py

```diff
@@ -212,44 +213,50 @@ def is_closing(self):
     async def async_close_cover(self, **kwargs):
         """Close the cover."""
 
-        if 'velocity' in kwargs:
-            velocity = kwargs['velocity']
-        else:
-            velocity = None
+        close_args = {
+            "wait_for_completion": False
+        }
 
-        if self.subtype is None:
-            await self.node.close(wait_for_completion=False, velocity=velocity)
-        else: 
-            await self.node.close(wait_for_completion=False, velocity=velocity, curtain=self.subtype)
+        if 'velocity' in kwargs and 'velocity' in inspect.getfullargspec(self.node.close).args:
+            close_args["velocity"] = kwargs["velocity"]
+
+        if self.subtype is not None:
+            close_args["curtain"] = self.subtype
+
+        await self.node.close(**kwargs)
 
     async def async_open_cover(self, **kwargs):
         """Open the cover."""
 
-        if 'velocity' in kwargs:
-            velocity = kwargs['velocity']
-        else:
-            velocity = None
+        open_args = {
+            "wait_for_completion": False
+        }
 
-        if self.subtype is None:
-            await self.node.open(wait_for_completion=False, velocity=velocity)
-        else:
-            await self.node.open(wait_for_completion=False, velocity=velocity, curtain=self.subtype)
+        if 'velocity' in kwargs and 'velocity' in inspect.getfullargspec(self.node.open).args:
+            open_args["velocity"] = kwargs["velocity"]
+
+        if self.subtype is not None:
+            open_args["curtain"] = self.subtype
+
+        await self.node.open(**kwargs)
```

Comment:
Actually this caused other issues now... I think adding velocity to DualRollerShutters is better option here. Velocity should be supported via function parameter 3 "fp3"
<img width="560" alt="image" src="https://github.com/pawlizio/my_velux/assets/15065886/082b4251-36e5-44d5-ae55-986c9c6c1887">